src/appleseed/renderer/modeling/volume/genericvolume.cpp

```diff
@@ -347,11 +347,11 @@ DictionaryArray GenericVolumeFactory::get_input_metadata() const
             .insert("type", "numeric")
             .insert("min",
                 Dictionary()
-                    .insert("value", "-0.9")
```

Comment:
Maybe we should set the limits for the UI to -1 and 1 and clamp to (-0.99, 0.99) where this value is actually used?
It's much nicer for application UIs in that way.